api/src/workers/base-event-handler.ts

```diff
@@ -41,6 +42,13 @@ abstract class BaseEventHandler {
       if (fromBlock > toBlock) {
         fromBlock = toBlock;
       }
+
+      // getEventsAtBlockHeightRange() has a block limit of 250 blocks. 
+      // If the range exceeds the limit of 200 then hard cap the block search range
+      if (fromBlock + this.stepSize < toBlock) {
+        toBlock = fromBlock + this.stepSize;
+      }
+
```

Comment:
Do you mind moving this change into a separate PR from the documentation updates? Best to keep it isolated.